master/buildbot/reporters/gerrit.py

```diff
@@ -175,6 +176,7 @@ def reconfigService(self, server, username, reviewCB=DEFAULT_REVIEW,
         self.summaryCB = summaryCB
         self.summaryArg = summaryArg
         self.builders = builders
+        self.gerrit_notify = notify
```

Comment:
while legacy code extensively uses public members, the new code is expected to be aware whether it's a part of the class' API or not.  I'd suggest to use `_gerrit_notify`